### ISSUE TITLE
Add more features apart from CURD .

### ISSUE THREAD

**Issue by Rick-mad-lab** [Rick-mad-lab/Todolist#8]

**Comment by sakshi135** [comment]
i want to contribute in this. please assign me this.

**Comment by Rick-mad-lab** [comment]
Introduce a new feature to me.

**Comment by aanchal-fatwani** [comment]
Hi @Rick-mad-lab, I'd like to work on this issue. I have an amazing feature in mind, to enable Voice capture from users instead of typing it out. 
Please assign me this issue to allow me to proceed with the same. Thanks.

**Comment by Rick-mad-lab** [comment]
nice we can work on this in a separate branch

**Comment by AshishisLiquid** [comment]
Add Time limit for Tasks, Maybe we can add notification for upcoming versions.

**Comment by Rick-mad-lab** [comment]
@AshishisLiquid yes we can do that or we can add more date and time per task too.
